Index: modules/jetty/src/main/java/org/mortbay/jetty/webapp/WebAppContext.java
===================================================================
--- modules/jetty/src/main/java/org/mortbay/jetty/webapp/WebAppContext.java	(revision 4733)
+++ modules/jetty/src/main/java/org/mortbay/jetty/webapp/WebAppContext.java	(working copy)
@@ -77,8 +77,9 @@
     
     private static String[] __dftConfigurationClasses =  
     { 
-        "org.mortbay.jetty.webapp.WebInfConfiguration", 
-        "org.mortbay.jetty.webapp.WebXmlConfiguration", 
+        "org.mortbay.jetty.webapp.WebInfConfiguration",
+        "org.mortbay.jetty.plus.webapp.EnvConfiguration",
+        "org.mortbay.jetty.annotations.Configuration",
         "org.mortbay.jetty.webapp.JettyWebXmlConfiguration",
         "org.mortbay.jetty.webapp.TagLibConfiguration" 
     } ;
