--- ./bin/jetty.sh	2009-10-02 00:45:47.000000000 -0700
+++ ./bin/hightide.sh	2009-10-14 17:30:39.000000000 -0700
@@ -11,7 +11,7 @@
 
 # Configuration files
 #
-# /etc/default/jetty
+# /etc/default/hightide
 #   If it exists, this is read at the start of script. It may perform any 
 #   sequence of shell commands, like setting relevant environment variables.
 #
@@ -31,7 +31,7 @@
 #
 #   The files will be checked for existence before being passed to jetty.
 #
-# $JETTY_HOME/etc/jetty.xml
+# $HIGHTIDE_HOME/etc/hightide.xml
 #   If found, used as this script's configuration file, but only if
 #   /etc/jetty.conf was not present. See above.
 #   
@@ -47,7 +47,7 @@
 # JAVA_OPTIONS
 #   Extra options to pass to the JVM
 #
-# JETTY_HOME
+# HIGHTIDE_HOME
 #   Where Jetty is installed. If not set, the script will try go
 #   guess it by first looking at the invocation path for the script,
 #   and then by looking in standard locations as $HOME/opt/jetty
@@ -56,7 +56,7 @@
 #
 #    <Arg><SystemProperty name="jetty.home" default="."/>/webapps/jetty.war</Arg>
 #
-# JETTY_PORT
+# HIGHTIDE_PORT
 #   Override the default port for Jetty servers. If not set then the
 #   default value in the xml configuration file will be used. The java
 #   system property "jetty.port" will be set to this value for use in
@@ -70,17 +70,17 @@
 #
 #    <Set name="Port">8080</Set>
 #
-# JETTY_RUN
-#   Where the jetty.pid file should be stored. It defaults to the
+# HIGHTIDE_RUN
+#   Where the hightide.pid file should be stored. It defaults to the
 #   first available of /var/run, /usr/var/run, and /tmp if not set.
 #  
-# JETTY_PID
-#   The Jetty PID file, defaults to $JETTY_RUN/jetty.pid
+# HIGHTIDE_PID
+#   The Jetty PID file, defaults to $HIGHTIDE_RUN/hightide.pid
 #   
-# JETTY_ARGS
+# HIGHTIDE_ARGS
 #   The default arguments to pass to jetty.
 #
-# JETTY_USER
+# HIGHTIDE_USER
 #   if set, then used as a username to run the server as
 #
 # Set to 0 if you do not want to use start-stop-daemon (especially on SUSE boxes)
@@ -136,10 +136,8 @@
 ##################################################
 # See if there's a default configuration file
 ##################################################
-if [ -f /etc/default/jetty6 ] ; then 
-  . /etc/default/jetty6
-elif [ -f /etc/default/jetty ] ; then 
-  . /etc/default/jetty
+if [ -f /etc/default/hightide ] ; then 
+  . /etc/default/hightide
 fi
 
 
@@ -162,28 +160,28 @@
 ##################################################
 # Jetty's hallmark
 ##################################################
-JETTY_INSTALL_TRACE_FILE="etc/jetty.xml"
+HIGHTIDE_INSTALL_TRACE_FILE="etc/hightide.xml"
 TMPJ=$TMP/j$$
 
 
 ##################################################
-# Try to determine JETTY_HOME if not set
+# Try to determine HIGHTIDE_HOME if not set
 ##################################################
-if [ -z "$JETTY_HOME" ] 
+if [ -z "$HIGHTIDE_HOME" ] 
 then
-  JETTY_HOME_1=`dirname "$0"`
-  JETTY_HOME_1=`dirname "$JETTY_HOME_1"`
-  if [ -f "${JETTY_HOME_1}/${JETTY_INSTALL_TRACE_FILE}" ] ; 
+  HIGHTIDE_HOME_1=`dirname "$0"`
+  HIGHTIDE_HOME_1=`dirname "$HIGHTIDE_HOME_1"`
+  if [ -f "${HIGHTIDE_HOME_1}/${HIGHTIDE_INSTALL_TRACE_FILE}" ] ; 
   then 
-     JETTY_HOME=${JETTY_HOME_1} 
+     HIGHTIDE_HOME=${HIGHTIDE_HOME_1} 
   fi
 fi
 
 
 ##################################################
-# if no JETTY_HOME, search likely locations.
+# if no HIGHTIDE_HOME, search likely locations.
 ##################################################
-if [ "$JETTY_HOME" = "" ] ; then
+if [ "$HIGHTIDE_HOME" = "" ] ; then
   STANDARD_LOCATIONS="           \
         /usr/share               \
         /usr/share/java          \
@@ -197,51 +195,45 @@
         /usr/local/share/java    \
         /home                    \
         "
-  JETTY_DIR_NAMES="              \
-        jetty-6                  \
-        jetty6                   \
-        jetty-6.*                \
-        jetty                    \
-        Jetty-6                  \
-        Jetty6                   \
-        Jetty-6.*                \
-        Jetty                    \
+  HIGHTIDE_DIR_NAMES="              \
+        hightide                    \
+        Hightide                    \
         "
         
-  JETTY_HOME=
+  HIGHTIDE_HOME=
   for L in $STANDARD_LOCATIONS 
   do
-     for N in $JETTY_DIR_NAMES 
+     for N in $HIGHTIDE_DIR_NAMES 
      do
-         if [ -d $L/$N ] && [ -f "$L/${N}/${JETTY_INSTALL_TRACE_FILE}" ] ; 
+         if [ -d $L/$N ] && [ -f "$L/${N}/${HIGHTIDE_INSTALL_TRACE_FILE}" ] ; 
          then 
-            JETTY_HOME="$L/$N"
+            HIGHTIDE_HOME="$L/$N"
          fi
      done
-     [ ! -z "$JETTY_HOME" ] && break
+     [ ! -z "$HIGHTIDE_HOME" ] && break
   done
 fi
 
 
 ##################################################
-# No JETTY_HOME yet? We're out of luck!
+# No HIGHTIDE_HOME yet? We're out of luck!
 ##################################################
-if [ -z "$JETTY_HOME" ] ; then
-    echo "** ERROR: JETTY_HOME not set, you need to set it or install in a standard location" 
+if [ -z "$HIGHTIDE_HOME" ] ; then
+    echo "** ERROR: HIGHTIDE_HOME not set, you need to set it or install in a standard location" 
     exit 1
 fi
 
-cd $JETTY_HOME
-JETTY_HOME=`pwd`
+cd $HIGHTIDE_HOME
+HIGHTIDE_HOME=`pwd`
 
 
 #####################################################
 # Check that jetty is where we think it is
 #####################################################
-if [ ! -r $JETTY_HOME/$JETTY_INSTALL_TRACE_FILE ] 
+if [ ! -r $HIGHTIDE_HOME/$HIGHTIDE_INSTALL_TRACE_FILE ] 
 then
-   echo "** ERROR: Oops! Jetty doesn't appear to be installed in $JETTY_HOME"
-   echo "** ERROR:  $JETTY_HOME/$JETTY_INSTALL_TRACE_FILE is not readable!"
+   echo "** ERROR: Oops! Jetty doesn't appear to be installed in $HIGHTIDE_HOME"
+   echo "** ERROR:  $HIGHTIDE_HOME/$HIGHTIDE_INSTALL_TRACE_FILE is not readable!"
    exit 1
 fi
 
@@ -256,15 +248,15 @@
     if [ -f $A ] 
     then
        CONF="$A" 
-    elif [ -f $JETTY_HOME/etc/$A ] 
+    elif [ -f $HIGHTIDE_HOME/etc/$A ] 
     then
-       CONF="$JETTY_HOME/etc/$A" 
+       CONF="$HIGHTIDE_HOME/etc/$A" 
     elif [ -f ${A}.xml ] 
     then
        CONF="${A}.xml" 
-    elif [ -f $JETTY_HOME/etc/${A}.xml ] 
+    elif [ -f $HIGHTIDE_HOME/etc/${A}.xml ] 
     then
-       CONF="$JETTY_HOME/etc/${A}.xml" 
+       CONF="$HIGHTIDE_HOME/etc/${A}.xml" 
     else
        echo "** ERROR: Cannot find configuration '$A' specified in the command line." 
        exit 1
@@ -284,14 +276,14 @@
 # but only if no configurations were given on the
 # command line.
 ##################################################
-if [ -z "$JETTY_CONF" ] 
+if [ -z "$HIGHTIDE_CONF" ] 
 then
   if [ -f /etc/jetty.conf ]
   then
-     JETTY_CONF=/etc/jetty.conf
-  elif [ -f "${JETTY_HOME}/etc/jetty.conf" ]
+     HIGHTIDE_CONF=/etc/jetty.conf
+  elif [ -f "${HIGHTIDE_HOME}/etc/jetty.conf" ]
   then
-     JETTY_CONF="${JETTY_HOME}/etc/jetty.conf"
+     HIGHTIDE_CONF="${HIGHTIDE_HOME}/etc/jetty.conf"
   fi
 fi
 
@@ -299,9 +291,9 @@
 # Read the configuration file if one exists
 ##################################################
 CONFIG_LINES=
-if [ -z "$CONFIGS" ] && [ -f "$JETTY_CONF" ] && [ -r "$JETTY_CONF" ] 
+if [ -z "$CONFIGS" ] && [ -f "$HIGHTIDE_CONF" ] && [ -r "$HIGHTIDE_CONF" ] 
 then
-  CONFIG_LINES=`cat $JETTY_CONF | grep -v "^[:space:]*#" | tr "\n" " "` 
+  CONFIG_LINES=`cat $HIGHTIDE_CONF | grep -v "^[:space:]*#" | tr "\n" " "` 
 fi
 
 ##################################################
@@ -313,7 +305,7 @@
   do
     if [ ! -r "$CONF" ] 
     then
-      echo "** WARNING: Cannot read '$CONF' specified in '$JETTY_CONF'" 
+      echo "** WARNING: Cannot read '$CONF' specified in '$HIGHTIDE_CONF'" 
     elif [ -f "$CONF" ] 
     then
       # assume it's a configure.xml file
@@ -330,11 +322,11 @@
          then
             CONFIGS="$CONFIGS $FILE" 
          else
-           echo "** WARNING: Cannot read '$FILE' specified in '$JETTY_CONF'" 
+           echo "** WARNING: Cannot read '$FILE' specified in '$HIGHTIDE_CONF'" 
          fi
       done
     else
-      echo "** WARNING: Don''t know what to do with '$CONF' specified in '$JETTY_CONF'" 
+      echo "** WARNING: Don''t know what to do with '$CONF' specified in '$HIGHTIDE_CONF'" 
     fi
   done
 fi
@@ -344,24 +336,24 @@
 #####################################################
 if [ -z "$CONFIGS" ] 
 then
-    CONFIGS="${JETTY_HOME}/etc/jetty-logging.xml ${JETTY_HOME}/etc/jetty.xml"
+    CONFIGS="${HIGHTIDE_HOME}/etc/hightide.xml"
 fi
 
 
 #####################################################
 # Find a location for the pid file
 #####################################################
-if [  -z "$JETTY_RUN" ] 
+if [  -z "$HIGHTIDE_RUN" ] 
 then
-  JETTY_RUN=`findDirectory -w /var/run /usr/var/run /tmp`
+  HIGHTIDE_RUN=`findDirectory -w /var/run /usr/var/run /tmp`
 fi
 
 #####################################################
 # Find a PID for the pid file
 #####################################################
-if [  -z "$JETTY_PID" ] 
+if [  -z "$HIGHTIDE_PID" ] 
 then
-  JETTY_PID="$JETTY_RUN/jetty.pid"
+  HIGHTIDE_PID="$HIGHTIDE_RUN/hightide.pid"
 fi
 
 
@@ -439,19 +431,19 @@
 JAVA_VERSION=`expr "$($JAVA -version 2>&1 | head -1)" : '.*1\.\([0-9]\)'`
 
 #####################################################
-# See if JETTY_PORT is defined
+# See if HIGHTIDE_PORT is defined
 #####################################################
-if [ "$JETTY_PORT" != "" ] 
+if [ "$HIGHTIDE_PORT" != "" ] 
 then
-  JAVA_OPTIONS="$JAVA_OPTIONS -Djetty.port=$JETTY_PORT"
+  JAVA_OPTIONS="$JAVA_OPTIONS -Djetty.port=$HIGHTIDE_PORT"
 fi
 
 #####################################################
-# See if JETTY_LOGS is defined
+# See if HIGHTIDE_LOGS is defined
 #####################################################
-if [ "$JETTY_LOGS" != "" ]
+if [ "$HIGHTIDE_LOGS" != "" ]
 then
-  JAVA_OPTIONS="$JAVA_OPTIONS -Djetty.logs=$JETTY_LOGS"
+  JAVA_OPTIONS="$JAVA_OPTIONS -Djetty.logs=$HIGHTIDE_LOGS"
 fi
 
 #####################################################
@@ -466,29 +458,29 @@
 #####################################################
 # Add jetty properties to Java VM options.
 #####################################################
-JAVA_OPTIONS="$JAVA_OPTIONS -Djetty.home=$JETTY_HOME -Djava.io.tmpdir=$TMP"
+JAVA_OPTIONS="$JAVA_OPTIONS -Djetty.home=$HIGHTIDE_HOME -Djava.io.tmpdir=$TMP"
 
-[ -f $JETTY_HOME/etc/start.config ] && JAVA_OPTIONS="-DSTART=$JETTY_HOME/etc/start.config $JAVA_OPTIONS"
+[ -f $HIGHTIDE_HOME/etc/start.config ] && JAVA_OPTIONS="-DSTART=$HIGHTIDE_HOME/etc/start.config $JAVA_OPTIONS"
 
 #####################################################
 # This is how the Jetty server will be started
 #####################################################
 
-JETTY_START=$JETTY_HOME/start.jar
-[ ! -f $JETTY_START ] && JETTY_START=$JETTY_HOME/lib/start.jar
+HIGHTIDE_START=$HIGHTIDE_HOME/start.jar
+[ ! -f $HIGHTIDE_START ] && HIGHTIDE_START=$HIGHTIDE_HOME/lib/start.jar
 
-RUN_ARGS="$JAVA_OPTIONS -jar $JETTY_START $JETTY_ARGS $CONFIGS"
+RUN_ARGS="$JAVA_OPTIONS -jar $HIGHTIDE_START $HIGHTIDE_ARGS $CONFIGS"
 RUN_CMD="$JAVA $RUN_ARGS"
 
 #####################################################
 # Comment these out after you're happy with what 
 # the script is doing.
 #####################################################
-#echo "JETTY_HOME     =  $JETTY_HOME"
-#echo "JETTY_CONF     =  $JETTY_CONF"
-#echo "JETTY_RUN      =  $JETTY_RUN"
-#echo "JETTY_PID      =  $JETTY_PID"
-#echo "JETTY_ARGS     =  $JETTY_ARGS"
+#echo "HIGHTIDE_HOME     =  $HIGHTIDE_HOME"
+#echo "HIGHTIDE_CONF     =  $HIGHTIDE_CONF"
+#echo "HIGHTIDE_RUN      =  $HIGHTIDE_RUN"
+#echo "HIGHTIDE_PID      =  $HIGHTIDE_PID"
+#echo "HIGHTIDE_ARGS     =  $HIGHTIDE_ARGS"
 #echo "CONFIGS        =  $CONFIGS"
 #echo "JAVA_OPTIONS   =  $JAVA_OPTIONS"
 #echo "JAVA           =  $JAVA"
@@ -499,22 +491,22 @@
 ##################################################
 case "$ACTION" in
   start)
-        echo -n "Starting Jetty: "
+        echo -n "Starting Hightide: "
 
         if [ "$NO_START" = "1" ]; then 
-	  echo "Not starting jetty - NO_START=1 in /etc/default/jetty6";
+	  echo "Not starting Hightide - NO_START=1 in /etc/default/jetty6";
           exit 0;
 	fi
 
 
 	if [ "$START_STOP_DAEMON" = "1" ] && type start-stop-daemon > /dev/null 2>&1
 	then
-          [ x$JETTY_USER = x ] && JETTY_USER=$(whoami)
-	  [ $UID = 0 ] && CH_USER="-c $JETTY_USER"
-	  if start-stop-daemon -S -p$JETTY_PID $CH_USER -d $JETTY_HOME -b -m -a $JAVA -- $RUN_ARGS 
+          [ x$HIGHTIDE_USER = x ] && HIGHTIDE_USER=$(whoami)
+	  [ $UID = 0 ] && CH_USER="-c $HIGHTIDE_USER"
+	  if start-stop-daemon -S -p$HIGHTIDE_PID $CH_USER -d $HIGHTIDE_HOME -b -m -a $JAVA -- $RUN_ARGS 
 	  then
 	      sleep 1
-	      if running $JETTY_PID
+	      if running $HIGHTIDE_PID
 	      then
                   echo OK
               else
@@ -524,63 +516,63 @@
 
 	else
 
-          if [ -f $JETTY_PID ]
+          if [ -f $HIGHTIDE_PID ]
           then            
-            if running $JETTY_PID
+            if running $HIGHTIDE_PID
             then
               echo "Already Running!!"
               exit 1
             else
               # dead pid file - remove
-              rm -f $JETTY_PID
+              rm -f $HIGHTIDE_PID
             fi
           fi
 
-          if [ x$JETTY_USER != x ] 
+          if [ x$HIGHTIDE_USER != x ] 
           then
-              touch $JETTY_PID
-              chown $JETTY_USER $JETTY_PID
-              su - $JETTY_USER -c "
+              touch $HIGHTIDE_PID
+              chown $HIGHTIDE_USER $HIGHTIDE_PID
+              su - $HIGHTIDE_USER -c "
                 $RUN_CMD &
                 PID=\$!
                 disown \$PID
-                echo \$PID > $JETTY_PID"
+                echo \$PID > $HIGHTIDE_PID"
           else
               $RUN_CMD &
               PID=$!
               disown $PID
-              echo $PID > $JETTY_PID
+              echo $PID > $HIGHTIDE_PID
           fi
 
-          echo "STARTED Jetty `date`" 
+          echo "STARTED Hightide `date`" 
         fi
 
         ;;
 
   stop)
-        echo -n "Stopping Jetty: "
+        echo -n "Stopping Hightide: "
 	if [ "$START_STOP_DAEMON" = "1" ] && type start-stop-daemon > /dev/null 2>&1; then
-	  start-stop-daemon -K -p $JETTY_PID -d $JETTY_HOME -a $JAVA -s HUP 
+	  start-stop-daemon -K -p $HIGHTIDE_PID -d $HIGHTIDE_HOME -a $JAVA -s HUP 
 	  sleep 1
-	  if running $JETTY_PID
+	  if running $HIGHTIDE_PID
 	  then
 	      sleep 3
-	      if running $JETTY_PID
+	      if running $HIGHTIDE_PID
 	      then
 		  sleep 30
-	          if running $JETTY_PID
+	          if running $HIGHTIDE_PID
 	          then
-	             start-stop-daemon -K -p $JETTY_PID -d $JETTY_HOME -a $JAVA -s KILL
+	             start-stop-daemon -K -p $HIGHTIDE_PID -d $HIGHTIDE_HOME -a $JAVA -s KILL
 		  fi
               fi
 	  fi
 
-	  rm -f $JETTY_PID
+	  rm -f $HIGHTIDE_PID
           echo OK
 	else
-	  PID=`cat $JETTY_PID 2>/dev/null`
+	  PID=`cat $HIGHTIDE_PID 2>/dev/null`
           TIMEOUT=30
-          while running $JETTY_PID && [ $TIMEOUT -gt 0 ]
+          while running $HIGHTIDE_PID && [ $TIMEOUT -gt 0 ]
           do
             kill $PID 2>/dev/null
             sleep 1
@@ -589,23 +581,23 @@
           
           [ $TIMEOUT -gt 0 ] || kill -9 $PID 2>/dev/null
 
-	  rm -f $JETTY_PID
+	  rm -f $HIGHTIDE_PID
           echo OK
 	fi
         ;;
 
   restart)
-        JETTY_SH=$0
-        if [ ! -f $JETTY_SH ]; then
-          if [ ! -f $JETTY_HOME/bin/jetty.sh ]; then
-            echo "$JETTY_HOME/bin/jetty.sh does not exist."
+        HIGHTIDE_SH=$0
+        if [ ! -f $HIGHTIDE_SH ]; then
+          if [ ! -f $HIGHTIDE_HOME/bin/jetty.sh ]; then
+            echo "$HIGHTIDE_HOME/bin/jetty.sh does not exist."
             exit 1
           fi
-          JETTY_SH=$JETTY_HOME/bin/jetty.sh
+          HIGHTIDE_SH=$HIGHTIDE_HOME/bin/jetty.sh
         fi
-        $JETTY_SH stop $*
+        $HIGHTIDE_SH stop $*
         sleep 5
-        $JETTY_SH start $*
+        $HIGHTIDE_SH start $*
         ;;
 
   supervise)
@@ -617,17 +609,17 @@
          ;;
 
   run|demo)
-        echo "Running Jetty: "
+        echo "Running Hightide: "
 
-        if [ -f $JETTY_PID ]
+        if [ -f $HIGHTIDE_PID ]
         then
-            if running $JETTY_PID
+            if running $HIGHTIDE_PID
             then
               echo "Already Running!!"
               exit 1
             else
               # dead pid file - remove
-              rm -f $JETTY_PID
+              rm -f $HIGHTIDE_PID
             fi
         fi
 
@@ -635,13 +627,13 @@
         ;;
 
   check)
-        echo "Checking arguments to Jetty: "
-        echo "JETTY_HOME     =  $JETTY_HOME"
-        echo "JETTY_CONF     =  $JETTY_CONF"
-        echo "JETTY_RUN      =  $JETTY_RUN"
-        echo "JETTY_PID      =  $JETTY_PID"
-        echo "JETTY_PORT     =  $JETTY_PORT"
-        echo "JETTY_LOGS     =  $JETTY_LOGS"
+        echo "Checking arguments to Hightide: "
+        echo "HIGHTIDE_HOME     =  $HIGHTIDE_HOME"
+        echo "HIGHTIDE_CONF     =  $HIGHTIDE_CONF"
+        echo "HIGHTIDE_RUN      =  $HIGHTIDE_RUN"
+        echo "HIGHTIDE_PID      =  $HIGHTIDE_PID"
+        echo "HIGHTIDE_PORT     =  $HIGHTIDE_PORT"
+        echo "HIGHTIDE_LOGS     =  $HIGHTIDE_LOGS"
         echo "CONFIGS        =  $CONFIGS"
         echo "JAVA_OPTIONS   =  $JAVA_OPTIONS"
         echo "JAVA           =  $JAVA"
@@ -649,9 +641,9 @@
         echo "RUN_CMD        =  $RUN_CMD"
         echo
         
-        if [ -f $JETTY_RUN/jetty.pid ]
+        if [ -f $HIGHTIDE_RUN/hightide.pid ]
         then
-            echo "Jetty running pid="`cat $JETTY_RUN/jetty.pid`
+            echo "Hightide running pid="`cat $HIGHTIDE_RUN/hightide.pid`
             exit 0
         fi
         exit 1
